odoo/tools/js_transpiler.py

```diff
@@ -227,7 +227,7 @@ def convert_variable_export_default(content):
     ^
     (?P<space>\s*)                      # space and empty line
     export\s*                           # export
-    (?P<object>{(\s*\w+\s*,?\s*)*}\s*)  # { a, b, c as x, ... }
+    (?P<object>{(\s*\w+\s*,?\s*)*})     # { a, b, c as x, ... }
```

Comment:
this pattern is also prone to catastrophic backtracking, for example this will trigger it:
```js
export {
    a,
    aReallyVeryLongName // that's long!
};
```

We may or may not want to support this (currently we don't), but it should not block the transpiler.

Cfr next comment for my rationale on the proposed alternative.

```suggestion
    (?P<object>{[\w\s,]+})     # { a, b, c as x, ... }
```